### ISSUE TITLE
Sound Balancing

### ISSUE THREAD

**Issue by emily-phet** [phetsims/ratio-and-proportion#59]
Keeping track of sound balancing thoughts here, for a bit later in the design process.

- When staccato ratio fitness sounds will be heard, probably the hand pickup/drop sounds should be in the background more. When no staccato ratio fitness sounds will be heard, pickup/drop should probably be foregrounded more.

**Comment by emily-phet** [comment]
Alternately, if no staccato ratio fitness sound will be heard, perhaps no pickup/drop sound should be heard at all.

**Comment by zepumph** [comment]
Unassigning until we are working on this.

**Comment by zepumph** [comment]
Now that we are working on #158, I think it is time to think about this issue. Over to @emily-phet for final thoughts and actionables.

**Comment by emily-phet** [comment]
That's happening in #158. Closing this issue. @zepumph let me know if you thought this issue was about something else.
